index.js

```diff
@@ -49,12 +49,21 @@ function fastifyView (fastify, opts, next) {
         return
       }
 
-      lru.set(page, engine.compile(html, options))
+      try {
+        lru.set(page, engine.compile(html, options))
+      } catch (error) {
+        that.send(error)
+        return
+      }
 
       if (!that.res.getHeader('content-type')) {
         that.header('Content-Type', 'text/html')
       }
-      that.send(lru.get(page)(data))
+      try {
+        that.send(lru.get(page)(data))
+      } catch (error) {
+        that.send(error)
+      }
```

Comment:
can you just wrap calling the function? The rest should not be wrapped.

Reply:
I tried but without that inside a try/catch I have the same a crash of the server ...
So I could update even this block in a similar way of the previous change.